crypto/src/test/java/org/springframework/security/crypto/bcrypt/BCryptTests.java

```diff
@@ -338,6 +338,11 @@ public void hashpwWorksWithOldRevision() {
 				"$2$05$......................bvpG2UfzdyW/S0ny/4YyEZrmczoJfVm");
 	}
 
+	@Test(expected = IllegalArgumentException.class)
+	public void hashpwFailsWhenSaltisTooShort() {
```

Comment:
Will you do me the favor of capitalizing "is" here? That way it looks like:

```suggestion
	public void hashpwFailsWhenSaltIsTooShort() {
```

Reply:
Changed function name.